postman_problems/solver.py

```diff
@@ -94,7 +94,7 @@ def cpp(edgelist_filename, start_node=None, edge_weight='distance', verbose=Fals
     g_odd_complete = create_complete_graph(odd_node_pairs_shortest_paths, flip_weights=True)
 
     logger_cpp.info('Find min weight matching using blossom algorithm')
-    odd_matching = dedupe_matching(nx.algorithms.max_weight_matching(g_odd_complete, True))
```

Comment:
why don't we need the deduping here anymore?

Reply:
the new version of max_weight_matching returns a set which does not contain duplicate.
https://networkx.org/documentation/stable/reference/algorithms/generated/networkx.algorithms.matching.max_weight_matching.html